libsolv-sys/src/export/pool.rs

```diff
@@ -3,6 +3,7 @@ use libc::{c_int, c_char};
 
 extern "C" {
     pub fn e_pool_id2solvable(pool: *const Pool, p: Id) -> *mut Solvable;
+    pub fn e_pool_solvable2id(pool: *const Pool, p: *const Solvable) -> Id;
     pub fn e_pool_solvid2str(pool: *mut Pool, p: Id) -> *const c_char;
     pub fn e_pool_match_nevr(pool: *mut Pool, s: *mut Solvable, d: Id) -> c_int;
     pub fn pool_whatprovides(pool: *mut Pool, d: Id) -> Id;
```

Comment:
I'm a little confused about why this one and the one below it don't have a prefix